src/main/java/org/apache/commons/cli/CommandLine.java

```diff
@@ -246,6 +331,44 @@ public String getOptionValue(char opt, String defaultValue)
     {
         return getOptionValue(String.valueOf(opt), defaultValue);
     }
+    
+    /**
+     * Retrieve the map of values associated to the option. This is convenient
+     * for options specifying Java properties like <tt>-Dparam1=value1
+     * -Dparam2=value2</tt>. The first argument of the option is the key, and
+     * the 2nd argument is the value. If the option has only one argument
+     * (<tt>-Dfoo</tt>) it is considered as a boolean flag and the value is
+     * <tt>"true"</tt>.
+     *
+     * @param option name of the option
+     * @return The Properties mapped by the option, never <tt>null</tt>
+     *         even if the option doesn't exists
+     * @since 1.4
+     */
+    public Properties getOptionProperties(Option option)
```

Comment:
I just copied the String variant of this including the javadoc, so I assume it was changed some time in the past and the java doc was not changed.